keras/src/saving/saving_api.py

```diff
@@ -1,18 +1,20 @@
 import os
 import zipfile
 
+import h5py
```

Comment:
Can you revert this for now? I think it so that `import keras` still works without `h5py` installed, which can be useful. The truth is that the custom handling logic of `h5py` should be replaced by a standard `LazyModule`.

Reply:
Done